network-api/networkapi/wagtailpages/pagemodels/buyersguide/utils.py

```diff
@@ -41,3 +41,10 @@ def sort_average(products):
     `products` is a QuerySet of ProductPages.
     """
     return sorted(products, key=lambda p: p.creepiness)
+
+
+def get_featured_cta(self):
```

Comment:
**Question** Have you considered adding a docstring to explain what this function does and why it needs a parameter?

Reply:
Updated!